Program/Project.py

```diff
@@ -0,0 +1,121 @@
+#for stars
+import random
+import java.awt.Font as Font
+
+def main():
+
+  #see if the image location can be found
+  try:
+    setMediaPath("C:\Users\mum\Desktop\Falmouth\comp120-tinkering-graphics-1")
```

Comment:
This code isn't portable because it references local resources in literals. When run from another location it results in a runtime error.